sdk/include/reactos/iernonce_undoc.h

```diff
@@ -0,0 +1,28 @@
+#ifndef _IERNONCE_UNDOC_H_
+#define _IERNONCE_UNDOC_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef VOID
+(CALLBACK *RUNONCEEX_CALLBACK)(
+    _In_ UINT CompleteCnt,
+    _In_ UINT TotalCnt,
+    _In_ DWORD_PTR dwReserved);
+VOID WINAPI
+    InitCallback(
+        _In_ RUNONCEEX_CALLBACK Callback,
+        _In_ BOOL bSilence);
+VOID WINAPI
+    RunOnceExProcess(
+        _In_ HWND hwnd,
+        _In_ HINSTANCE hInst,
+        _In_ LPCSTR pszCmdLine,
+        _In_ int nCmdShow);
```

Comment:
```suggestion

VOID WINAPI
InitCallback(
    _In_ RUNONCEEX_CALLBACK Callback,
    _In_ BOOL bSilence);

VOID WINAPI
RunOnceExProcess(
    _In_ HWND hwnd,
    _In_ HINSTANCE hInst,
        _In_ LPCSTR pszCmdLine,
        _In_ int nCmdShow);
```
Standard formatting...

Reply:
corrected now!

Reply:
No idea why github still shows the broken formatting with extra indentations....

Reply:
hope it looks good now. seems I have to find sometime to give my editor a lesson 😡